api/HTMLMediaElement.json

```diff
@@ -2799,9 +2804,21 @@
             "edge": {
               "version_added": "17"
             },
-            "firefox": {
-              "version_added": false
-            },
+            "firefox": [
```

Comment:
FYI, added this since you saw it last time. There is another property `sinkId` which was added at the same time as setSinkId to Firefox. You can see it in searchfox here: https://searchfox.org/mozilla-central/source/dom/webidl/HTMLMediaElement.webidl#278 (and I also see when I run code in FF nightly)

So I've mirrored the support info for that too.